Source/Bogus/Faker[T].cs

```diff
@@ -246,10 +247,10 @@ private void PopulateInternal(T instance, string[] ruleSets)
                     }
                 }
 
-                foreach( var ruleSet in ruleSets )
+                foreach (var ruleSet in ruleSets)
```

Comment:
Hey @gdimauro , no worries, I'm going to rebase and squash the commit history in this PR so we have just 1 commit with only the necessary changes. Thanks again for getting your PR this far; so far it looks wonderful and very much easier for me to digest.

I'm just making a few minor changes. I'll CC you the PR once I'm done reviewing your changes.